biosdk-services/src/main/resources/application.properties

```diff
@@ -26,3 +26,5 @@ server.tomcat.accesslog.className=io.mosip.kernel.core.logger.config.SleuthValve
 management.endpoint.health.show-details=always
 management.endpoints.web.exposure.include=info,health,refresh,env
 
+mosip.biosdk.log-reqeuest-response-enabled=true
```

Comment:
by default lets not keep this enabled, since these are sentive information

Reply:
Currently, the property cannot be changed outside using the config file, as cloud config is not yet enabled and tested. So for our current testing it is enabled. When we will externalize with cloud config it we shall keep it disabled.